src/network/message.rs

```diff
@@ -412,6 +414,11 @@ impl Decodable for RawNetworkMessage {
             payload,
         })
     }
+
+    #[inline]
+    fn consensus_decode<D: io::Read>(d: D) -> Result<Self, encode::Error> {
```

Comment:
Get rid of. Since we're using `consensus_decode` on all inner-types, they will enforce the limits there.